solutions/7.js

```diff
@@ -0,0 +1,12 @@
+const solution = (num, i = 1, bigD = -1) => {
+  if (i < num) {
+    if (num % i == 0) {
```

Comment:
You can do `num % == 0 && bigD < i` and take out the line with `if (bigD < i)`. Just to reduce the number of line of code.

Reply:
why do you need to check if `bigD < 1 `?

Reply:
I am not checking if `bigD` is less than 1. Aren't we checking to see if `bigD` is less than the current index, `i`? If it is, then we reassign the value of `bigD` to `i`, right?